private-templates-service/client/src/NavBar.tsx

```diff
@@ -11,41 +11,54 @@ import {
   UncontrolledDropdown,
   DropdownToggle,
   DropdownMenu,
-  DropdownItem } from 'reactstrap';
-import '@fortawesome/fontawesome-free/css/all.css';
+  DropdownItem
+} from "reactstrap";
+import "@fortawesome/fontawesome-free/css/all.css";
 
-function UserAvatar(props) {
+function UserAvatar(props: any) {
```

Comment:
While implicit typing is nice, I find a good habit to get into is having functions return an explicit type for safety (for the next person who comes along and tries to modify it to return a number for example, they will know that other places expect it to be a component)

`function UserAvatar(props: any): Component`